lib/main.js

```diff
@@ -0,0 +1,16 @@
+"use 6to5";
+let NVAtom = require("./NVAtom");
+
+exports.activate = (state) => {
+  atom.nvatom = new NVAtom(state);
```

Comment:
How frowned upon is this? Is there a better way to get access to my app instance [like I need to do here](https://github.com/jonmagic/nvatom-package/pull/1/files#diff-0b6887f0de76a95f247b16342e6390a3R63)? /cc @nathansobo @maxbrunsfeld @kevinsawicki